static/elements/chromedash-feature.html

```diff
@@ -381,18 +388,46 @@
       return implStatusChrome === 'Deprecated' ||
              implStatusChrome === 'No longer pursuing';
     },
+    _computePreLaunch: function(implStatusChrome) {
+      return implStatusChrome === 'No active development' ||
+             implStatusChrome === 'Proposed' ||
+             implStatusChrome === 'In development' ||
+             implStatusChrome === 'Behind a flag' ||
+             implStatusChrome === 'In experimental framework';
+    },
     _computeEditLinkHidden: function(feature) {
       return '/admin/features/edit/' + feature.id;
     },
-    _computeCrbug: function(link) {
+    _computeCrbugNumber: function(link) {
       if (!link) {
-        return link;
+        return;
       }
       var m = link.match(/[\/|?id=]([0-9]+)$/); // pull out bug number.
       if (m) {
-        return '#' + m[1];
+        return m[1];
+      }
+    },
+    _computeCrbug: function(link) {
+      var CrBugNumber = this._computeCrbugNumber(link)
+      if (CrBugNumber) {
+        return '#' + CrBugNumber;
       }
       return link;
+    },
+    _computeNewBugUrl: function(feature) {
+      var url = "https://bugs.chromium.org/p/chromium/issues/entry";
+      var params = [];
+      if (feature.bug_components.length) {
+        params.push("components=" + feature.bug_components.map(encodeURIComponent));
```

Comment:
Suggest using just `Blink` if no components are supplied - such bugs should be triaged quicker (one less queue to go through).